source/AI.cpp

```diff
@@ -3292,20 +3292,26 @@ void AI::MovePlayer(Ship &ship, const PlayerInfo &player, Command &activeCommand
 			if(shift)
 				ship.SetTargetShip(shared_ptr<Ship>());
 
-			double closest = numeric_limits<double>::infinity();
+			double best = numeric_limits<double>::infinity();
 			bool foundEnemy = false;
 			bool foundAnything = false;
+			bool distancePriority = Preferences::Has("Board target");
+			double agility = ship.Acceleration() * ship.TurnRate();
```

Comment:
Why would I care about the agility by itself (the agility value should already be included in the costs of the ship)?

Reply:
Solved by the std::function in lambda